assets/airports/egnm.json

```diff
@@ -10,251 +10,2015 @@
+                [53.676908400, -1.544641800],
+                [53.678007300, -1.545086100],
+                [53.679118400, -1.545433100],
+                [53.680238700, -1.545681900],
+                [53.681365000, -1.545831800],
+                [53.682494400, -1.545882500],
+                [53.683624000, -1.545833800],
+                [53.684750300, -1.545685800],
+                [53.685870700, -1.545438900],
+                [53.686982100, -1.545093700],
+                [53.688081200, -1.544651100],
+                [53.689165300, -1.544112400],
+                [53.690231400, -1.543478800],
+                [53.691276500, -1.542752200],
+                [53.692297800, -1.541934300],
+                [53.693292600, -1.541027400],
+                [53.694258100, -1.540033900],
+                [53.695191600, -1.538956400],
+                [53.696090500, -1.537797700],
+                [53.696952500, -1.536561100],
+                [53.697775100, -1.535249800],
+                [53.698556000, -1.533867400],
+                [53.699293100, -1.532417500],
+                [53.699984300, -1.530904100],
+                [53.700627600, -1.529331500],
+                [53.701221200, -1.527703800],
+                [53.701763500, -1.526025500],
+                [53.702253000, -1.524301300],
+                [53.702688200, -1.522535800],
+                [53.703067900, -1.520734000],
+                [53.703391000, -1.518900800],
+                [53.703656700, -1.517041300],
+                [53.703864100, -1.515160600],
+                [53.704012600, -1.513264100],
+                [53.704102000, -1.511356900],
+                [53.704131800, -1.509444400]
+            ]
+        }
+    ],  
     "runways": [
         {
             "name": ["14", "32"],
             "end": [
-                ["N53d52m17.15", "W001d40m08.68"],
-                ["N53d51m37.31", "W001d39m07.44"]
+                ["N53d52m17.15", "W001d40m08.68", "674ft"],
+                ["N53d51m37.31", "W001d39m07.44", "662ft"]
             ],
-            "ils": [false, true]
+            "ils": [true, true],
+            "glideslope": [3.50, 3.00]
         }
     ],
     "sids": {
-        "POL": {
-            "icao": "POL",
-            "name": "Pole Hill Two Xray",
-            "suffix": {"14":"2X"},
+        "DOPEK": {
+            "icao": "DOPEK",
+            "name": "Dopek",
+            "suffix": {"14":"2X", "32":"2W"},
             "rwy": {
-                "14": [["D082N", "A30"], ["D082J", "A35"]]
+                "14": ["_ILBF2DME", "_INTGAM327", ["_GAM327036", "A30+|A70-"], ["_GAM327032", "A35+|A70-"], ["_GAM327029", "A70"], "_GAM327019", "_GAM327015", "_GAM327014", "_GAM327POL118"],
+                "32": [["_ILF0.5DME", "A12+"], "_ILF2.1DME", "_ILF3.5DME", ["_INTGAM319", "A30+|A70-"], ["_GAM319038", "A35+|A70-"], ["_GAM319035", "A60+|A70-"], ["_GAM319031", "A70"], "_GAM319019", "_GAM319018", "_GAMR319INTPOLR118", "_GAM15DME", "DOPEK"]
```

Comment:
- `_ILF3.5DME` should probably be fly-over
- the `A70-`'s aren't needed because of the restriction at 31DME GAM